.infrastructure/pvc.yml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: PersistantVolumeClaim
+metadata:
+  name: pvc-todoapp
+  namespace: todoapp
+spec:
+  storageClassName: standart
+  volumeName: pv-todoapp
+  accessModes:
+    - ReadWriteMany
+  resouces:
+    storage: 1Gi
```

Comment:
The 'resources' field should have a 'requests' subfield to specify the requested storage. It should look like:
resources:
  requests:
    storage: 1Gi